testing/sdk_auth.py

```diff
@@ -150,6 +150,8 @@ def get_download_stream(url: str) -> requests.Response:
         log.error("%s", e)
         raise RuntimeError("Failed to download the keytab file: {}".format(repr(e)))
 
+    log.info("Downloaded %d bytes from %s to %s", os.stat(output_filename).st_size, keytab_url, output_filename)
```

Comment:
We should probably be consistent with string formatting (`"%s"` vs `"{}".format()`) but it's not critical.

Reply:
Yeah I just did it this way on account of the surrounding file doing it